app_dart/test/request_handlers/reset_prod_task_test.dart

```diff
@@ -49,6 +53,33 @@ void main() {
         ),
         sha: '7d03371610c07953a5def50d500045941de516b8',
       );
+      startedBuild = const Build(
```

Comment:
If these are const, can they be instantiated as static const outside of setup?

Reply:
done